docs/internal/test-requirements/TestRequirements.md

```diff
@@ -29,13 +29,6 @@ R00500| IDN labels (names) should be valid.
 R00600| Non-IDN ASCII labels (names) should be valid.                                      |[RFC1123] [RFC2782]  |[BASIC00]
 R00700| A DNS zone should have a parent zone from which it is delegated.                   |                     |[BASIC01]
 R00800| A DNS zone should have at least one accessible name server that hosts it.          |                     |[BASIC02]
-R00900| A name server for a zone should respond on a query.                                |                     |[BASIC04]
-R01000| A name server for a zone should respond with SOA record on SOA query.              |[RFC2181]            |[BASIC04] [DELEGATION06]
-R01100| A name server for a zone should respond with RCODE NoError on SOA query.           |                     |[BASIC04]
-R01200| A name server for a zone should respond with AA flag set on SOA query.             |[RFC2181]            |[BASIC04]
-R01300| A name server for a zone should respond with NS RRset on NS query.                 |[RFC2181]            |[BASIC04]
-R01400| A name server for a zone should respond with RCODE NoError on NS query.            |                     |[BASIC04]
-R01500| A name server for a zone should respond with AA flag set on NS query.              |[RFC2181]            |[BASIC04]
```

Comment:
Should we really remove these requirements line? Maybe we should only remove/update the related testcase.

Reply:
Good point. I updated so that BASIC04 is replaced with both CONNECTIVITY01 and CONNECTIVITY02.